students/daniel_carrasco/session05/mailroom.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+from datetime import date
+import os
+import sys
+
+def main():
+    while True:
+        VALID_RESPONSES = (1,2,3,4)
+        try:
+            user_choice = int(input('\nChoose an action(1 - 4):\n\n\
+                    1 - Send a Thank You to a single donor.\n\
+                    2 - Create a Report.\n\
+                    3 - Send letters to all donors.\n\
+                    4 - Quit\n'))
+            arg_dict = {1:thankyou, 2:report, 3: letter,4:quit}
+            if arg_dict[user_choice] == 'quit':
+                sys.exit()
+            else:
+                arg_dict[user_choice]()
+        except ValueError:
+            print("Input must be an integer, try again.")
+        except KeyError:
+            print('Choice must be a menu input (1-4)')
+        continue
+
+def thankyou():
```

Comment:
Try modifying the code so that new donations, as opposed to only new donors, can be entered.